### DISCUSSION TITLE
PC端下载的是1.1.28版本打开还是1.1.27版，另外看影视提示服务器找不到请求的资源怎么办？

### DISCUSSION THREAD

**Discussion by fuxin-mail** [xiaohucode/yidaRule#214] [General]
/
/
![微信图片_20240812171036](https://github.com/user-attachments/assets/b1d629ba-e0b0-4eb1-ad5e-5feef601aef4)

**Comment by falling7down** [comment]


**Reply by fuxin-mail** [reply]
PC版本是不是只能导入本地源？不能网络导入？我这个打开网络导入就显示这样的
![屏幕截图 2024-08-12 201005](https://github.com/user-attachments/assets/7c13ebf6-2b49-4b80-96e7-714f0545956f)

**Reply by chenxingmoonset** [reply]
> PC版本是不是只能导入本地源？不能网络导入？我这个打开网络导入就显示这样的
> 
> ![屏幕截图 2024-08-12 201005](https://github.com/user-attachments/assets/7c13ebf6-2b49-4b80-96e7-714f0545956f)
> 
> 

可以网络导入

**Reply by fuxin-mail** [reply]
我也成功了谢谢各位

**Reply by maomaovip** [reply]
怎么成功的啊？

**Reply by fuxin-mail** [reply]
最后没办法了，我就猛点导入突然就提示成功了哈哈

**Reply by maomaovip** [reply]
好吧
